root/usr/share/tancredi/data/templates/nethesis-firmware.tmpl

```diff
@@ -0,0 +1,15 @@
+{%- if provisioning_user_agent matches '/Nethesis NP-X5/' -%}
```

Comment:
The regexp matches a substring: is it the expected behavior? Or do we need an anchor?

```suggestion
{%- if provisioning_user_agent matches '/^Nethesis NP-X5/' -%}
```